### ISSUE TITLE
MKS wifi printing goes wrong

### ISSUE THREAD

**Issue by Peto21** [makerbase-mks/MKS-TFT#418]
If I put a gcode on the SD and then print it goes perfectly. But if I send a gcode from Cura via WiFi and then want to print from the TFT, it all goes wrong. The printer will run in the air somewhere and both the bed and the hotend will not heat up. I use the latest version of Cura and updated the MKS with the latest version

**Comment by fireflylr** [comment]
What version of firmware are you using now? Is it the latest firmware?
--mks

**Comment by Peto21** [comment]
I updated several times with MKS-TFT35 V1.0.7. I got it from here https://github.com/makerbase-mks/MKS-TFT/tree/master/MKS-TFT3.5 But the screen of the TFT remains 1.0.6. So whether it's 1.0.6 or 1.0.7?

**Comment by Peto21** [comment]
After some start-up problems and the great help of Markerbase, it really works perfectly now. I am happy with this choice. Thanks for the great help Markerbase!

**Comment by daniellezama** [comment]
I have the same problem i have instaled the last versión an make the same thing print un the air. Could You share the solución?

**Comment by Peto21** [comment]
> 
> 
> I have the same problem i have instaled the last versión an make the same thing print un the air. Could You share the solución?

Install the latest Cura 4.4 plugin. This should fix the problem. If that doesn't help either, send a message to Markerbase, they really helped me. After sending emails back and forth a bit they sent me a new plugin (4.4) and the problems were solved. It still works perfectly here. I am really happy with the MKS stuff.

**Comment by daniellezama** [comment]
Hi Thanks for the answer I am using 4.4 cura plugin in a 4.5 Cura version, with 1.0.7 Firmware for the 3.5 MKS TFT Screen. The first job runs great. Second Job make this strange thing. I am using marlin 2.0 bug fix in a MKS SGENL V1.0. I Have tested the Marlin firmware with a reprap 12864 Full graphics smart controller and runs Great. Is not the marlin.  I am using 57600 Bauds in both hardwares in the tft and motherboard.

This thing is very strange. Must shut down the printer before a new job for use the tft screen.

How can email to Makerbase?

Regards

Daniel

**Comment by daniellezama** [comment]
Well after trying the other versions of firmwares (change config settings disable babasteps) and change bauds configuration too ( i have tested in 57600 112500 an 250000) could conclude 
--> version 1.05 stable but no gcode window and no tft image when printing) ,
-->version 1.06 stable with gcode window and tft image when printing, but when send the program remotely not show the print image. 
-->version  1.07 unstable show the print image and run fine only first printing. next prints in the air. Also problems for detect the sd ( must shut down and restart for read the SD).

Finally my choice version 1.0.6 less problems I have.

Waiting for version 1.08 (Hope MKS finally fix the problems)

Regards

**Comment by Peto21** [comment]
I am using cura 4.6.1, MKS 4.4 plugin for cura and 1.0.7 Firmware for the 3.5 MKS TFT Screen. And it works with the MKS Robin motherboard and baud rate 250000. And it really works perfectly. I suddenly had problems last week. But after some searching it turned out to be due to the SD card. After I replaced it everything worked fine again.

I now see below the message "Closed" I pressed the wrong button. Sorry!
